server/routes/getAllTodos.js

```diff
@@ -4,8 +4,9 @@ const hyper = connect(process.env.HYPER)
 
 module.exports = async function getAllTodos (req, res) {
   try {
-    const result = await hyper.data.list()
-    res.json(result)
+    const { docs } = await hyper.data.list()
+    const todos = docs.filter(item => item.type === 'todo')
```

Comment:
Instead of fetching all documents, you can use `query` to fetch documents that match search criteria. You can `hyper.data.query({ type: "todo" })` instead of manually filtering them.